### ISSUE TITLE
Uncaught Error: Class 'SimpleXMLElement' not found in accept-sample-app/getToken.php:24

### ISSUE THREAD

**Issue by collinanderson** [AuthorizeNet/accept-sample-app#25]
Is there a required library that I need to install or something?

**Comment by brianmc** [comment]
What version of PHP are you using?

**Comment by collinanderson** [comment]
7.0 (Ubuntu 16.04)

**Comment by brianmc** [comment]
You may need to install simplexml on ubuntu http://stackoverflow.com/questions/35593521/php-7-simplexml

**Comment by collinanderson** [comment]
Yeah, I eventually figured out I need to `sudo apt install php-xml`, but you might want to document that somewhere.

**Comment by the94air** [comment]
That is totally true. I was struggling for hours because of this. You can make our life easier by putting a warning or someting in the documentation, please. Thank you.

**Comment by eramudeep** [comment]
i am already having the php-xml 
still getting the error
